src/board/game.mjs

```diff
@@ -0,0 +1,31 @@
+// functions that handle the evaluations of the game
+function handleWinner() {
+  const container = document.querySelector('.grid-container');
+  const winnerHeader = document.createElement('h2');
+  winnerHeader.classList.add('winner-header');
+  winnerHeader.innerHTML = 'You win!';
+  container.after(winnerHeader);
+}
+
+function markTileCorrect(tileIndex) {
+  const tile = document.querySelector(`.color-tile-${tileIndex}`);
+  // TODO: style this class later
+  tile.classList.add('u-tile-correct');
+}
+
+function checkForWin(currentGrid, answerKey) {
+  let numberOfCorrectTiles = 0;
+  for (let i = 0; i < answerKey.length; i += 1) {
+    if (answerKey[i] === currentGrid[i]) {
+      numberOfCorrectTiles += 1;
+      markTileCorrect(i);
+    }
+  }
+  if (numberOfCorrectTiles === currentGrid.length) {
+    handleWinner();
+  }
+}
```

Comment:
love how small this file is, and how small the functions are, its very readable code!